integration-tests/src/test/java/org/web3j/protocol/core/CoreIT.java

```diff
@@ -207,27 +271,44 @@ public void testEthGetUncleCountByBlockHash() throws Exception {
     public void testEthGetUncleCountByBlockNumber() throws Exception {
         EthGetUncleCountByBlockNumber ethGetUncleCountByBlockNumber =
                 web3j.ethGetUncleCountByBlockNumber(
-                        DefaultBlockParameter.valueOf("latest")).send();
+                        DefaultBlockParameter.valueOf(new BigInteger("232"))).send();
```

Comment:
test will fail if node has not synced block 232 - how likely is this? better make the test bulletproof